src/test/ui/issues/issue-81220.stderr

```diff
@@ -0,0 +1,25 @@
+error[E0609]: no field `c` on type `Foo`
+  --> $DIR/issue-81220.rs:26:20
+   |
+LL |   let test = fooer.c;
+   |                    ^
+   |                    |
+   |                    unknown field
+   |                    help: try to use the whole path to the field: `fooer.bar.c`
```

Comment:
Applying this suggestion will result in

```rust
let test = fooer.fooer.bar.c;
```

which is not what we want. We want `fooer.first.bar.c` instead.

Reply:
There was a logic error in the code, should be fixed now.